src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java

```diff
@@ -77,7 +77,7 @@ public ClaimantResponseContentProvider(
                 content.put("defendantAdmissionAccepted", "I accept this amount");
                 ResponseAcceptation responseAcceptation = (ResponseAcceptation) claimantResponse;
                 content.putAll(responseAcceptationContentProvider.createContent(responseAcceptation));
-                content.put("formaliseOption", responseAcceptation.getFormaliseOption().getDescription());
+                content.put("formaliseOption", responseAcceptation.getFormaliseOption().get().getDescription());
```

Comment:
`Optional.get()` without checking its presence. Replace your `Optional.get()` calls with `orElse` or `orElseThrow` baed on the scenario.